collector/impl/prometheus_metrics_collector.go

```diff
@@ -31,6 +31,9 @@ import (
 	"github.com/hawkular/hawkular-openshift-agent/http"
 	"github.com/hawkular/hawkular-openshift-agent/log"
 	"github.com/hawkular/hawkular-openshift-agent/prometheus"
+	"github.com/hawkular/hawkular-openshift-agent/util/expand"
+	"os"
+	"strings"
```

Comment:
Updated. Do you know of any check style plugins for golang? just so this will obvious to any future committers.